### ISSUE TITLE
When showing GeoJSON "description" on feature info, also provide a "Raw Data" option.

### ISSUE THREAD

**Issue by kring** [TerriaJS/terriajs#2826]
If a GeoJSON file has a `description` property, its value is shown on the feature info instead of the usual table of properties. #1452 suggests we should change this behavior (i.e. use a different property name other than `description`), but sometimes it's useful. In any case, we should provide a "Show Raw Data" in this scenario, just as we do in the case of a feature info template, so that the user can get back to the full property table.

**Comment by chloeleichen** [comment]
This is no longer an issue @kring ? It doesn't show `description` atm but it does show the table

**Comment by kring** [comment]
Yeah pretty sure it's still an issue. We're you testing with a geojson with a property named `description`?

**Comment by nf-s** [comment]
In v8 we have the opposite problem - `description` is being dropped and all properties are being shown
